syncano/models/fields.py

```diff
@@ -704,18 +742,51 @@ def validate(self, value, model_instance):
             except (ValueError, TypeError):
                 raise SyncanoValueError('Expected an object')
 
-        if not isinstance(value, GeoPointStruct):
-            raise SyncanoValueError('Expected an GeoPointStruct')
+        if not isinstance(value, GeoPoint):
```

Comment:
This kind of checks can be done with assertions like that https://wiki.python.org/moin/UsingAssertionsEffectively:
`assert isinstance(value, GeoPoint), 'Expected GeoPoint'`